src/foam/blob/Blob.js

```diff
@@ -401,7 +398,10 @@ foam.CLASS({
         if ( o2 == null ) return 1;
         if ( o2 == this ) return 0;
         return foam.util.SafetyUtil.compare(getId(), o2.getId());
-      `
+      `,
+      code: function compareTo(other) {
```

Comment:
believe this should be 
```suggestion
      code: function(other) {
```